Sources/Preferences/PreferencesWindowController.swift

```diff
@@ -74,15 +74,18 @@ public final class PreferencesWindowController: NSWindowController {
 	///
 	/// - See `close()` to close the window again.
 	/// - See `showWindow(_:)` to show the window without the convenience of activating the app.
-	/// - Note: Unless you need to open a specific pane, prefer not to pass a parameter at all
-	/// - Parameter preferencePane: Identifier of the preference pane to display.
+	/// - Note: Unless you need to open a specific pane, prefer not to pass a parameter at all or `nil`.
+	/// - Parameter preferencePane: Identifier of the preference pane to display, or `nil` to show the
+	///   tab that was open when the user last closed the window.
 	public func show(preferencePane preferenceIdentifier: PreferencePaneIdentifier? = nil) {
```

Comment:
```suggestion
	public func show(preferencePane preferencePaneIdentifier: PreferencePaneIdentifier? = nil) {
```

?

I think we should be consistent in using `preferencePane` over just `preference`.